src/style.css

```diff
@@ -1 +1,94 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.body {
+  margin: 0;
+  padding: 0 8px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Here you should specify the fallback font-families. The correct usage would be `font-family: Avenir, Arial, Helvetica, sans-serif;` if `Avenir` is the name you've defined in the `@font-face` rule.